docs/integrations/pack-docs.md

```diff
@@ -55,3 +73,195 @@ It is possible to add an image placeholder which links to an external video. For
 [![Video Name](https://img.youtube.com/vi/[YOUTUBE_VIDEO_ID]/0.jpg)](https://www.youtube.com/watch?v=[YOUTUBE_VIDEO_ID] "Video Name")
 ```
 An example of including a YouTube video can be seen at the [Zoom Feed README](https://github.com/demisto/content/blob/master/Packs/FeedZoom/README.md).
+
+## Pack Keywords, Tags, Use-cases & Categories
+In order classify packs and make them easier to find, there are a number of pack metadata elements that can be used:
+- Use-case -- a name of a situation in which packs can be used - usually the pack name. 
+
+ Examples: Impossible Traveler, PCAP Analysis
+
+- Tags -- words from the same semantic field of the use-case - they help find packs using filters or the search bar, and are visible on the screen to help understand what the pack is all about.
+
+ Examples: GDPR, PII, TIM
+
+- Keywords -- like tags, but these don’t show in the UI.
+
+For example, for a pack related to messaging, you may want to add "msg" as a keyword
```

Comment:
Please elaborate a bit more, for example `you may want to add "msg" as a keyword so your pack will appear in search results`

Reply:
Added